### PR TITLE
Create a test script to load test the POST call and add instructions to README

### PR DESCRIPTION
**Pull request by Kinshuk1993**
- Create a test script to load test the POST call via APIC

Doc how to
- Run the load script
- Watch the MQ queue grow
- Stop the load script
- Watch the MQ queue shrink

### CODE REVIEW

File: EventEnabledInsurance/build/pipeline.yaml [hunk 1]
```diff
@@ -200,6 +200,7 @@ spec:
         - name: git-source
           workspace: git-source
 
+    # push to apic
```
Comment (DanRoseus) [inline EventEnabledInsurance/build/pipeline.yaml:203]: No need to change anything, but I don't see the point in comments that say the same thing as the code in the line below. Personally I would only put in comments that actually explain things not already understandable from the naming

File: EventEnabledInsurance/post-load-test.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    exit 1
+}
+
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
+all_done="\xF0\x9F\x92\xAF"
+info="\xE2\x84\xB9"
+CURRENT_DIR=$(dirname $0)
+NAMESPACE="cp4i"
+TARGET_POST_CALLS=1000
```
Comment (DanRoseus) [inline EventEnabledInsurance/post-load-test.sh:39]: 1000 sounds like a lot. I guess this takes 15 to 20 mins for DB Writer to catch up and empty the queue? How long does it take for this script to run with 1000?

I'm wondering if maybe we should add a parameter to specify this so it's easy to change.

File: EventEnabledInsurance/post-load-test-readme.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Overview
+This load test script will add 1000 messages to the MQ Queue `Quote` by making REST calls via IBM API Connect. These messages will start appearing in the MQ queue named `Quotes` and the IntegrationServer `ace-db-writer-int-srv-eei` will read from the queue, one message per second and add them to the [System Of Record database](readme.md#working-directly-with-the-system-of-record-database).
```
Comment (DanRoseus) [inline EventEnabledInsurance/post-load-test-readme.md:2]: Docs look good. I was expecting this to be added to the end of the readme, but admittedly the readme is getting out of control. Maybe include a link to this from the readme.md? Maybe later we can decide if we want to split the other stories out of the readme as well.

**Review by DanRoseus — CHANGES_REQUESTED** [review]

File: EventEnabledInsurance/build/pipeline.yaml (hunk 1 above)
Comment (Kinshuk1993) [reply EventEnabledInsurance/build/pipeline.yaml]: Done

File: EventEnabledInsurance/post-load-test-readme.md (hunk 3 above)
Comment (Kinshuk1993) [reply EventEnabledInsurance/post-load-test-readme.md]: Done

**Pull request merged**